python/sglang/srt/model_executor/model_runner.py

```diff
@@ -197,6 +199,7 @@ def __init__(
                 "use_mla_backend": self.use_mla_backend,
             }
         )
+        set_use_cpu(global_server_args_dict["device"] == "cpu")
```

Comment:
Move this to server_args::__post_init__, https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/server_args.py#L248